src/app/app.css

```diff
@@ -245,6 +245,42 @@ html, body, #root {
   overflow: auto;
 }
 
+.dashboard-card-resizable-wrapper { 
+  display: flex;
+  justify-content: space-between 
```

Comment:
I think we can remove this. We just need the `display: flex` to make the resize bar expanded.